hooks/pre-push

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python3
+import sys, os
+import re
+import string
+import subprocess
+
+args = sys.argv
+
+# get branch name
+bashCommand = 'git branch'
+process = subprocess.Popen(bashCommand.split(), stdout=subprocess.PIPE)
+output, error = process.communicate()
+branch_name = output[2:-1]
+
+# get list of pushed files
+get_list_of_files ='git diff --name-only ' + args[1] + '/' + branch_name.decode('utf-8')
+process = subprocess.Popen(get_list_of_files.split(), stdout=subprocess.PIPE)
+output, error = process.communicate()
+output = output.decode('utf-8')
+
+# checking .py files
+for file_name in output.split():
+	if bool(re.match(r'.*\.py$', file_name)):
```

Comment:
there might be python files without .py extension.
Just FYI, because finding such files is not a trivial task